packages/utils/src/types/status.ts

```diff
@@ -0,0 +1,6 @@
+export enum Status {
```

Comment:
Just "Status"? Looks suspicious. What about "MintingStatus", something easier for SDK users.